react/src/lib/components/DeckGLMap/layers/terrain/map3DLayer.ts

```diff
@@ -255,19 +273,22 @@ export interface Map3DLayerProps<D> extends ExtendedLayerProps<D> {
     // Url to png image representing the height mesh.
     mesh: string;
 
-    // Bounding box of the terrain mesh, [minX, minY, maxX, maxY] in world coordinates
-    bounds: [number, number, number, number];
+    // Horizontal extent of the terrain mesh.
+    frame?: Frame;
+
+    // Deprecated: mesh's horizonal extent as [xmin, ymin, xmax, ymax]
+    bounds?: [number, number, number, number];
 
     // Mesh error in meters. The output mesh is in higher resolution (more vertices) if the error is smaller.
     meshMaxError: number;
 
     // Url to png image for map properties. (ex, poro or perm values as a texture)
     propertyTexture: string;
 
-    // Rotates map counterclockwise in degrees around 'rotPoint'.
+    // Rotates map counterclockwise in degrees around 'rotPoint' specified below.
     rotDeg: number;
```

Comment:
Not urgent, but I think we should move also `rotDeg` and `rotPoint` to the `frame` prop.

Reply:
Can I do that in next pull request?